packages/tasks_panel/src/misc/cap-e2e-config.ts

```diff
@@ -0,0 +1,102 @@
+import { RelativePattern, Uri, commands } from "vscode";
+import { ChildProcessWithoutNullStreams, spawn } from "child_process";
+import { getLogger } from "../../src/logger/logger-wrapper";
+import {
+  CAP_DEPLOYMENT_CONFIG,
+  LabelType,
+  ProjectInfo,
+  ProjectTypes,
+  addTaskDefinition,
+  areResourcesReady,
+  generateMtaDeployTasks,
+  isFileExist,
+  isTasksSettled,
+  waitForResource,
+} from "./e2e-config";
+import { last } from "lodash";
+
+export interface CapProjectConfigInfo extends ProjectInfo {
+  type: string;
+}
+
+/* istanbul ignore next */
+function invokeCommand(config: { cmd: string; args: string[]; cwd: string }, additionalArgs: string[]): Promise<void> {
+  function executeSpawn(
```

Comment:
does `executeSpawn` belongs in this `cap-e2e-config.ts` file?
It seems very generic...

Reply:
reviewed and N/A